.golangci.yml

```diff
@@ -5,6 +5,8 @@ run:
 linters-settings:
   govet:
     check-shadowing: true
+  gofmt:
+    simplify: true
```

Comment:
Please note that this don't change the behaviour as this is already the default value. This just states this again.